jcstress-core/src/main/java/org/openjdk/jcstress/Options.java

```diff
@@ -39,14 +39,15 @@
 import java.io.PrintStream;
 import java.text.SimpleDateFormat;
 import java.util.*;
-import java.util.concurrent.TimeUnit;
 
 /**
  * Options.
  *
  * @author Aleksey Shipilev (aleksey.shipilev@oracle.com)
  */
 public class Options {
+
+    private static final String LIST_OPTION_DESCRIPTION="List the available tests matching the requested settings, after all filters (like CPU count) are applied. In verbose mode it prints all real combinations which will run.";
```

Comment:
feel free to split this in two lines if you prefer, you are right on the readability aspect, anyways the concatenation would happen just once at the application startup

Reply:
thanx! Split.